modules/messages/src/weights_ext.rs

```diff
@@ -341,7 +344,7 @@ pub trait WeightInfoExt: WeightInfo {
 		Self::receive_single_message_proof().saturating_sub(Self::receive_single_prepaid_message_proof())
 	}
 
-	/// Returns pre-dispatch weight of single message delivery callback call.
+	/// Returns pre-dispatch weight of single callback call.
```

Comment:
```suggestion
	/// Returns pre-dispatch weight of single callback call.
	///
	/// When benchmarking the weight please take into consideration both the `OnMessageAccepted` and `OnDeliveryConfirmed` callbacks. The method should return the greater of the two, because it's used to estimate the weight in both contexts.
```